npm/src/index.js

```diff
@@ -36,7 +36,7 @@ export const firstTimeExperienceSetup = async () => {
 
   let platform = ''
   process.argv.slice(2).forEach(arg => {
-    if (arg === 'setup') {
+    if (arg === 'setup' || arg === 'build') {
```

Comment:
What is the reason for this change? I think it might trigger an FTE run for every build.
This isn't good if the user has elected to install android and not windows - We don't want to prompt them to install windows deps again.